src/lib/game/handler.js

```diff
@@ -27,9 +27,10 @@ class GameHandler extends Socket {
   }
 
   // Connects to given host through given port
-  connect(host, port) {
+  connect(host, realm) {
+    this.realm=realm;
```

Comment:
I've fixed it, however i suggest you to make a "confirm and squash" before accept this PR

https://help.github.com/articles/about-pull-request-merge-squashing/